setup/project_task_pull_request/odoo/addons/project_task_pull_request

```diff
@@ -0,0 +1 @@
+../../../../project_task_pull_request
```

Comment:
should this file be removed from the commit?

Reply:
No, they are needed for pip